desktop.css

```diff
@@ -1,248 +1,706 @@
+}
+
+.lists li a {
+  font-size: 30px;
+  color: #fff;
+  text-decoration: none;
+}
+
+.open-menu .lists {
+  transform: translateX(0);
+  z-index: 5;
+}
+
+#logo {
+  font-size: 25px;
+  color: #6070ff;
+  text-decoration: none;
+  font-weight: 900;
+}
+
+.content {
+  background: no-repeat center/100%, url('./img/header-shapes mobile.svg');
+  background-color: #fff;
+  background-size: contain;
+  padding: 24px;
+  border-radius: 0 0 0 60px;
+  min-height: 700px;
+}
+
+.sub-content {
+  margin-top: 95px;
+  color: #172b4d;
+}
+
+.sub-content h1,
+h2,
+p {
+  padding-top: 10px;
+  font-size: 16px;
+  line-height: 24px;
+  font-weight: 400;
+}
+
+.social-media {
+  margin-top: 10px;
+}
+
+footer p {
+  color: #ebebff;
+  font-size: 20px;
+  font-weight: 400;
+  text-align: center;
+  margin-top: 15px;
+  animation: 3s slidein;
+}
+
+@keyframes slidein {
+  from {
+    margin-left: 100%;
+    width: 300%;
   }
 
-  .canopy-card {
-    background: #e5e5e5;
+  to {
+    margin-left: 0%;
     width: 100%;
   }
+}
 
-  .canopy-card-section,
-  .canopy-second-card-section,
-  .canopy-third-card-section,
-  .canopy-fourth-card-section {
-    height: 90%;
-    width: 76%;
-    left: 0;
-    top: 0;
-    border-radius: 22px;
-    background-color: white;
-    display: flex;
-    justify-content: center;
-    cursor: pointer;
-    align-items: center;
-  }
+.social-media p {
+  color: #7f8cff;
+  text-transform: uppercase;
+  font-size: 16px;
+}
 
-  .canopy-fourth-card-section:hover {
-    border: 2px solid #dfe1e6;
-  }
+.sub-content h1,
+h2 {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  width: 88%;
+}
 
-  .canopy-card-section:hover {
-    border: 2px solid #dfe1e6;
-  }
+ul {
+  padding-top: 10px;
+  max-width: 160px;
+  list-style-type: none;
+  display: flex;
+  justify-content: space-between;
+}
 
-  .canopy-second-card-section:hover {
-    border: 2px solid #dfe1e6;
-  }
+/* -------------POPUP WINDOW------------ */
+
+.popup {
+  display: none;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100vh;
+  z-index: 1000;
+  background-color: #c1c7d073;
+}
 
-  .canopy-third-card-section:hover {
-    border: 2px solid #a7aeff;
-  }
+.popup ::-webkit-scrollbar {
+  width: 2px;
+  visibility: hidden;
+}
 
-  .snapshoot-portfolio {
-    width: 50%;
-    height: 100%;
+.popup.active {
+  display: inline-block;
+}
+
+.popup-content {
+  position: fixed;
+  left: 50%;
+  top: 0;
+  transform: translateX(-50%);
+  height: 95%;
+  min-width: 340px;
+  width: 45%;
+  background-color: #fff;
+  transition: top 250ms linear, opacity 200ms linear;
+  display: flex;
+  flex-direction: column;
+  overflow-x: hidden;
+  overflow-y: scroll;
+  border-radius: 16px;
+  padding: 18px;
+}
+
+.popup-content.active {
+  opacity: 1;
+  top: 2% !important;
+}
+
+.popup-h2 {
+  color: #172b4d;
+  font-size: 28px;
+  margin: 0;
+  padding: 0;
+}
+
+.popup-title img {
+  width: 306px;
+  height: 220px;
+  margin: 12px 0;
+}
```

Comment:
- [ ] The popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1), What if we give the 100% width to your popup images 😄 

- Your Implementation:

![image png](https://user-images.githubusercontent.com/79058364/170547335-d58e4da4-dbf5-40c1-9811-4588a2221805.png)

Reply:
Hello @BenMukebo , thanks for these feedbacks. My apologies for those mistakes I did, I am trying my best to implement the major changes as the time is gonna be up.

Thanks

Reply:
@omarbabou, I'm sure you can do it 💪🏻 😄

Reply:
Hello @omarbabou, I'm sure you can do it 💪🏻 😄 Good luck 👍🏻

Reply:
Yes I can do it but the problem is the time and in addition they approved that of my codding partner while we were working on the same project.